package.json

```diff
@@ -111,7 +108,9 @@
     "clean:install": "npm run clean && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
     "compile": "npm run compile:clean && npm run compile:build",
-    "compile:build": "lerna ls | sed 's/@cerner\\///g' | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
+    "compile:build": "npm run compile:carbon-graphs && npm run compile:terra-graphs",
+    "compile:carbon-graphs": "babel packages/carbon-graphs/src --config-file babel.config.js --out-dir packages/carbon-graphs/lib --copy-files",
+    "compile:terra-graphs": "babel packages/terra-graphs/src --config-file babel.config.js --out-dir packages/terra-graphs/lib --copy-files",
```

Comment:
the terra-graphs-docs repo should also be compiled.

Reply:
Why are we changing this?

Reply:
Oh, I was mistaken that terra-graphs-docs won't be compiled. Since that's the case, I reverted the changes here:
c632cd0a4b8418441db684b2d06783bac979c224 

the `-all` flag makes it so that private packages can also be compiled.